backend/src/main/java/net/vidux/camhub/discovery/ViduxHelperWrapper.java

```diff
@@ -0,0 +1,44 @@
+package net.vidux.camhub.discovery;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+class ViduxHelperWrapper {
+  private static final String[] FIND_HIKVISION_IP_CAMERAS_COMMAND = {
+    "vidux-helper", "system", "findHikvisionIPCameras"
+  };
+  private ApplicationContext applicationContext =
+      new AnnotationConfigApplicationContext(TimeOutConfiguration.class);
+  private int timeOut = applicationContext.getBean(TimeOutConfiguration.class).getTimeOut();
```

Comment:
Egy darab konstanst behúzni, azért ennél könnyebben is lehet! Nem tudom mi volt itt az előzmény, de ez így nagyon bonyolult teljesen feleslegesen!